### PR TITLE
ansible-test - Add alpine 3.19

### LINKED ISSUE #82019: Add Alpine 3.19 to ansible-test
### Summary

This is a remote VM and container addition.

### Issue Type

Feature Idea

### Component Name

`ansible-test`

### PR DESCRIPTION
**Pull request by Akasurde**
##### SUMMARY

Fixes: #82019

Signed-off-by: Abhijeet Kasurde <akasurde@redhat.com>

##### ISSUE TYPE
- Bugfix Pull Request

### CODE REVIEW

**Comment by webknjaz** [comment]
I suppose the CI fails because of a missing image that is yet to be provided, right?

**Comment by Akasurde** [comment]
> I suppose the CI fails because of a missing image that is yet to be provided, right?

Yes

**Comment by webknjaz** [comment]
The recent failure `Remote Alpine 3.19 - 6` excerpt:
```console
01:01 Installing requirements for Python 3.11 (controller)
01:01 Stream command with data: /usr/bin/python3.11
01:01 Execute command: /usr/bin/python3.11 /tmp/ansible-test-yj5b_nll-pip.py install --disable-pip-version-check coverage==7.3.2
01:02 error: externally-managed-environment
01:02
01:02 × This environment is externally managed
01:02 ╰─>
01:02     The system-wide python installation should be maintained using the system
01:02     package manager (apk) only.
01:02
01:02     If the package in question is not packaged already (and hence installable via
01:02     "apk add py3-somepackage"), please consider installing it inside a virtual
01:02     environment, e.g.:
01:02
```
_(https://dev.azure.com/ansible/ansible/_build/results?buildId=100879&view=logs&j=33c146a7-663f-5b6e-305f-1c2308bc22a4&t=f6b13fa7-a283-5711-7e41-6473a856ee00&l=156)_

Related PEP: https://peps.python.org/pep-0668/.

Perhaps, we need to update the controller provisioning in `ansible-test`.

**Comment by Akasurde** [comment]
@webknjaz Yes, @mattclay is working on the required changes from `ansible-test` side. Thanks.

**Comment by mattclay** [comment]
@Akasurde I've opened https://github.com/ansible/ansible/pull/82527 to resolve the PEP 668 issues that occur when ansible-test installs requirements. The changes do not extend to integration tests themselves, so any integration tests performing pip installs outside a virtual environment are likely to fail. We'll need to look at each of those to see how we want to approach them. Hopefully they can simply be converted to use virtual environments.

**Comment by webknjaz** [comment]
`ansible-test-container` is failing under Alpine 3.19. I restarted it just in case, to see if it's flaky.

**Comment by webknjaz** [comment]
Though, I see now that there's been 3 attempts already, all failing. So it's something to be fixed in the PR.

**Comment by webknjaz** [comment]
@Akasurde try rebasing? The CI is still failing on podman tests.

**Comment by Akasurde** [comment]
@webknjaz @mattclay is working on some changes in ansible-test to accommodate the changes for cgroup v2 + alpine 3.19. I will wait for those changes to merge first to pass these CI tests. Thanks.

File: test/lib/ansible_test/_data/completion/docker.txt [hunk 1]
```diff
@@ -2,6 +2,7 @@ base image=quay.io/ansible/base-test-container:6.1.0 python=3.12,3.7,3.8,3.9,3.1
 default image=quay.io/ansible/default-test-container:9.4.0 python=3.12,3.7,3.8,3.9,3.10,3.11 context=collection
 default image=quay.io/ansible/ansible-core-test-container:9.4.0 python=3.12,3.7,3.8,3.9,3.10,3.11 context=ansible-core
 alpine3 image=quay.io/ansible/alpine3-test-container:6.3.0 python=3.11 cgroup=none audit=none
+alpine319 image=quay.io/ansible/alpine319-test-container:main python=3.11 cgroup=none audit=none
```
Comment (mattclay) [inline test/lib/ansible_test/_data/completion/docker.txt]:
```suggestion
alpine319 image=quay.io/ansible/alpine319-test-container:7.1.0 python=3.11 cgroup=none audit=none
```

**Review by mattclay — APPROVED** [review]

**Pull request merged**